pkg/cmd/dcos.go

```diff
@@ -158,48 +144,10 @@ func newPluginCommand(ctx api.Context, cmd plugin.Command, isDummyCoreCommand bo
 	}
 }
 
-func customHelpCommand(ctx api.Context, root *cobra.Command, plugins []*plugin.Plugin) *cobra.Command {
-	return &cobra.Command{
-		Use:   "help [command]",
-		Short: "Help about any command",
-		Long: `Help provides help for any command in the application.
-		Simply type ` + root.Name() + ` help [path to command] for full details.`,
-		RunE: func(c *cobra.Command, args []string) error {
-			cmd, remArgs, e := c.Root().Find(args)
-			if cmd == nil || e != nil {
-				c.Printf("Unknown help topic %#q\n", args)
-				c.Root().Usage()
-			} else {
-				for _, p := range plugins {
-					for _, pluginCmd := range p.Commands {
-						if cmd.Name() == pluginCmd.Name {
-							// We don't support global plugins right now so no plugin should be run without a cluster.
-							_, err := ctx.Cluster()
-							if err != nil {
-								ctx.Logger().Error("Error: no cluster is attached")
-								return nil
-							}
-
-							args := []string{pluginCmd.Name}
-							args = append(args, remArgs...)
-							args = append(args, "--help")
-							invokePlugin(ctx, pluginCmd, args)
-							return nil
-						}
-					}
-				}
-				cmd.InitDefaultHelpFlag() // make possible 'help' flag to be shown
-				cmd.Help()
-			}
-			return nil
-		},
-	}
-
-}
-
 // extractCorePlugin extracts the bundled core plugin into the plugins folder.
 func extractCorePlugin(ctx api.Context, cluster *config.Cluster) (*plugin.Plugin, error) {
-	ctx.Logger().Warn(`Extracting "dcos-core-cli"...`)
+	ctx.Logger().Warn(`Extracting "dcos-core-cli"...
+This setup is deprecated, see https://docs.mesosphere.com/1.12/cli/experiments/#automatic-installation-of-core-and-enterprise-cli-plugins for more information.`)
```

Comment:
I put this URL for now but we should have a new section for 1.13 as it's not an experiment anymore.